plugins/flutter_aepcore/android/src/main/java/com/adobe/marketing/mobile/flutter/flutter_aepcore/FlutterAEPCoreDataBridge.java

```diff
@@ -118,6 +124,36 @@ static String stringFromPrivacyStatus(final MobilePrivacyStatus privacyStatus) {
         return AEP_PRIVACY_STATUS_UNKNOWN;
     }
 
+    static InitOptions initOptionsFromMap(final Object option) {
+        if (!(option instanceof Map)) {
+            return null;
+        }
+
+        Map optionsAsMap = (Map) option;
+
+        Map<String, Object> initOptionMap = getNullableMap(optionsAsMap, "initOptions");
+        if (initOptionMap == null) {
+            return null;
+        }
+        String appId = getNullableString(initOptionMap, APPID_KEY);
+        String filePath = getNullableString(initOptionMap, FILE_PATH_KEY);
+        Boolean lifecycleAutomaticTrackingEnabled = getNullableBoolean(initOptionMap, LIFECYCLE_AUTOMATION_TRACKING_KEY);
+        Map<String, String> lifecycleAdditionalContextData = getNullableMap(initOptionMap, LIFECYCLE_ADDITIONAL_CONTEXTDATA_KEY);
```

Comment:
This cast is incorrect. Can you add `getNullableStringMap()` which verifies the values are all String?